### PR TITLE
DDCE-5225

### PR DESCRIPTION
**Pull request by MRBeresford**

### CODE REVIEW

**Review by frikit — CHANGES_REQUESTED** [review]

File: build.sbt [hunk 1]
```diff
@@ -6,15 +6,16 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 
 lazy val appName: String = "register-estate-hub-frontend"
 
-lazy val root = (project in file("."))
+ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / majorVersion := 0
+
+lazy val microservice = (project in file("."))
   .enablePlugins(PlayScala, SbtDistributablesPlugin, SbtSassify)
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply build.sbt]: removed

File: build.sbt [hunk 2]
```diff
@@ -6,15 +6,16 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
...
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
     DefaultBuildSettings.scalaSettings,
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply build.sbt]: removed and removed import

File: build.sbt [hunk 3]
```diff
@@ -6,15 +6,16 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
...
     DefaultBuildSettings.defaultSettings(),
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply build.sbt]: removed and removed import

File: build.sbt [hunk 4]
```diff
@@ -6,15 +6,16 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
...
     inConfig(Test)(testSettings),
-    majorVersion := 0,
-    scalaVersion := "2.13.12",
     name := appName,
     RoutesKeys.routesImport += "models._",
     TwirlKeys.templateImports ++= Seq(
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply build.sbt:18]: added

File: project/AppDependencies.scala [hunk 5]
```diff
@@ -2,26 +2,26 @@ import sbt.*
...
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-30"  % mongoVersion,
     "org.scalatestplus"           %% "scalacheck-1-17"          % "3.2.18.0",
     "org.jsoup"                   %  "jsoup"                    % "1.17.2",
     "org.scalatest"               %% "scalatest"                % "3.2.18",
-    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.30",
-    "org.wiremock"                %  "wiremock-standalone"      % "3.4.2",
+    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.31",
+    "org.wiremock"                %  "wiremock-standalone"      % "3.5.4",
     "wolfendale"                  %% "scalacheck-gen-regexp"    % "0.1.2",
     "com.vladsch.flexmark"        %  "flexmark-all"             % "0.64.8"
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply project/AppDependencies.scala]: deleted

File: build.sbt [hunk 6]
```diff
@@ -6,15 +6,16 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 
 lazy val appName: String = "register-estate-hub-frontend"
 
-lazy val root = (project in file("."))
+ThisBuild / scalaVersion := "2.13.12"
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply build.sbt]: updated

File: project/AppDependencies.scala [hunk 7]
```diff
@@ -2,26 +2,26 @@ import sbt.*
...
-    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  % mongoVersion,
+    "uk.gov.hmrc"                 %% "bootstrap-test-play-30"   % boostrapVersion,
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-30"  % mongoVersion,
     "org.scalatestplus"           %% "scalacheck-1-17"          % "3.2.18.0",
     "org.jsoup"                   %  "jsoup"                    % "1.17.2",
     "org.scalatest"               %% "scalatest"                % "3.2.18",
-    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.30",
-    "org.wiremock"                %  "wiremock-standalone"      % "3.4.2",
+    "org.mockito"                 %% "mockito-scala-scalatest"  % "1.17.31",
+    "org.wiremock"                %  "wiremock-standalone"      % "3.5.4",
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply project/AppDependencies.scala]: deleted

File: project/AppDependencies.scala [hunk 8]
```diff
@@ -2,26 +2,26 @@ import sbt.*
...
   )
 
   private lazy val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                 %% "bootstrap-test-play-28"   % boostrapVersion,
-    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  % mongoVersion,
+    "uk.gov.hmrc"                 %% "bootstrap-test-play-30"   % boostrapVersion,
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-30"  % mongoVersion,
     "org.scalatestplus"           %% "scalacheck-1-17"          % "3.2.18.0",
     "org.jsoup"                   %  "jsoup"                    % "1.17.2",
     "org.scalatest"               %% "scalatest"                % "3.2.18",
```
[the remark this answers is not in the document]
Comment (MRBeresford) [reply project/AppDependencies.scala]: deleted

File: build.sbt [hunk 9]
```diff
@@ -1,20 +1,18 @@
 import play.sbt.routes.RoutesKeys
 import sbt.Def
 import scoverage.ScoverageKeys
-import uk.gov.hmrc.DefaultBuildSettings
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
```
Comment (frikit) [inline build.sbt:4]: delete this

File: project/plugins.sbt [hunk 10]
```diff
@@ -1,18 +1,11 @@
...
-  "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
-)
-
-addSbtPlugin("uk.gov.hmrc"          % "sbt-auto-build"          % "3.20.0")
+addSbtPlugin("uk.gov.hmrc"          % "sbt-auto-build"          % "3.21.0")
 addSbtPlugin("uk.gov.hmrc"          % "sbt-distributables"      % "2.5.0")
-addSbtPlugin("com.typesafe.play"    % "sbt-plugin"              % "2.8.20")
-addSbtPlugin("org.scalastyle"       %% "scalastyle-sbt-plugin"  % "1.0.0")
-addSbtPlugin("org.scoverage"        % "sbt-scoverage"           % "2.0.9")
-addSbtPlugin("io.github.irundaia"   % "sbt-sassify"             % "1.5.2")
```
Comment (frikit) [inline project/plugins.sbt:15]: add back sbt-sassify not sure it affects builds package

**Review by frikit — CHANGES_REQUESTED** [review]

**Review by frikit — APPROVED** [review]

**Pull request merged**